### PR TITLE
[RateLimiter] Allow configuration value "no_limit"

### PR DESCRIPTION
**Pull request by Nyholm**
| Q             | A
| ------------- | ---
| Branch?       | 5.x
| Bug fix?      | maybe?
| New feature?  | not sure
| Deprecations? | 
| Tickets       | 
| License       | MIT
| Doc PR        | 

I dont see any reason why we should allow people to configure "no_limit". I assume this was just forgotten.

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -1846,14 +1846,14 @@ private function addRateLimiterSection(ArrayNodeDefinition $rootNode)
...
-                                        ->values(['fixed_window', 'token_bucket', 'sliding_window'])
+                                        ->values(['fixed_window', 'token_bucket', 'sliding_window', 'no_limit'])
                                     ->end()
                                     ->integerNode('limit')
                                         ->info('The maximum allowed hits in a fixed interval or burst')
                                         ->isRequired()
                                     ->end()
                                     ->scalarNode('interval')
-                                        ->info('Configures the fixed interval if "strategy" is set to "fixed_window" or "sliding_window". The value must be a number followed by "second", "minute", "hour", "day", "week" or "month" (or their plural equivalent).')
+                                        ->info('Configures the fixed interval if "strategy" is set to "fixed_window", "sliding_window" or "no_limit". The value must be a number followed by "second", "minute", "hour", "day", "week" or "month" (or their plural equivalent).')
```
Comment (wouterj) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: This is not true, `no_limit` doesn't have any options (as it blindly accepts everything you throw at it)

**Review by wouterj — APPROVED** [review]
Thanks! I indeed didn't intentionally forbid this.

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (Nyholm) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: Oops. Sorry

**Review by chalasr — APPROVED** [review]

**Comment by chalasr** [comment]
Thank you Tobias.

**Pull request merged**